src/main/thrift/parquet.thrift

```diff
@@ -883,13 +928,42 @@ struct ColumnChunk {
 
   /** Encrypted column metadata for this chunk **/
   9: optional binary encrypted_column_metadata
+  /** 
+    * The column order for this chunk.
+    * 
+    * If not set readers should check FileMetadata.column_orders
+    * instead.
+    *
+    * Populated in both PAR1 and PAR3
+    */
+  10: optional ColumnOrder column_order
```

Comment:
I'm afraid this might complicate page index because we have to check consistency of ColumnOrder across row groups. At the moment we have only one ColumnOrder, which is not a big issue. It may have problem if we introduce more orders in the future. This order is important to guide us on how to interpret serialized min_value/max_values in the statistics. Perhaps we can put this in the `SchemaElement` instead?

Reply:
Schema element sounds OK to me we can also leave as is, I think I copied this from #242

Reply:
I'll move.

Reply:
I think maybe just keeping as is make sense actually, so I'll remove this field and we can keep the list or move that to a DATA_PAGE

Reply:
different collations are a not-uncommon idea in databases (e.g. case insensitive comparison).  One could argue on whether this should be modeled as a different logical type, a requirement on clients on how they store the data or using this concept but it is not unreasonable to model it in this way?

Reply:
here is #221 (follow on to #196)